src/main/res/values-v21/styles-v21.xml

```diff
@@ -0,0 +1,7 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <style name="SimpleToolbar">
+        <item name="android:theme">@style/ThemeOverlay.AppCompat.Dark.ActionBar</item>
+        <item name="android:background">?attr/colorPrimary</item>
+        <item name="android:elevation">4dp</item>
+    </style>
+</resources>
```

Comment:
You'll notice I named this `styles-v21` and not just `styles`. That name is completely arbitrary, but **the fact that it is different is not arbitrary.**

If I used `values-v21/styles.xml`, then this would overwrite all of our styles, and we'd have to include all the styles in the new file as well. Like this, it'll only override the single style that it needs to.